components/standings/commons/standings_parse_wiki.lua

```diff
@@ -176,4 +179,29 @@ function StandingsParseWiki.makeScoringFunction(tabletype, args)
 	error('Unknown table type')
 end
 
+---@param args table
+---@param tableType StandingsTableTypes
+---@return StandingsTiebreaker[]
+function StandingsParseWiki.parseTiebreakers(args, tableType)
+	local tiebreakerInput = Json.parseIfString(args.tiebreakers) or {}
+	local tiebreakers = {}
+	for _, tiebreaker in ipairs(tiebreakerInput) do
+		table.insert(tiebreakers, TiebreakerFactory.tiebreakerFromName(tiebreaker))
+	end
+	if #tiebreakers == 0 then
+		if tableType == 'ffa' then
+			tiebreakers = {
+				TiebreakerFactory.tiebreakerFromName('points'),
+				TiebreakerFactory.tiebreakerFromName('manual'),
+			}
+		elseif tableType == 'swiss' then
+			tiebreakers = {
+				TiebreakerFactory.tiebreakerFromName('matchdiff'),
+				TiebreakerFactory.tiebreakerFromName('manual'),
+			}
```

Comment:
[oos] potentionally add option for custom defaults (maybe via Module:Info)?

i know for the old stuff we can solve it easily via the wrappers, but i think it might be neat to allow this as it would save inputs for quite a few events on several wikis

Reply:
Maybe, we can think about that for later